<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginPage</class>
 <widget class="QDialog" name="LoginPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>663</width>
    <height>464</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="Background">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>663</width>
     <height>464</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>203</x>
     <y>94</y>
     <width>257</width>
     <height>318</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 19px;
background-color: rgba(217, 217, 217, 0.6);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>266</x>
     <y>323</y>
     <width>143</width>
     <height>41</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel, QPushButton {
    background-color: white;
    border-radius: 19px;
    font-weight: bold;
    font-family: &quot;Roboto&quot;;
    color: red;
    font-size: 18px;
}

QLabel:hover, QPushButton:hover {
    background-color: red;
    color: white;
}
</string>
   </property>
   <property name="text">
    <string>Login</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>269</x>
     <y>381</y>
     <width>79</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 15px;
font-weight: semibold;
font-family: &quot;Poppins&quot;;
padding-up: 4px;

</string>
   </property>
   <property name="text">
    <string>Not a User?</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>352</x>
     <y>383</y>
     <width>66</width>
     <height>23</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel, QPushButton {
    background-color: red;
    border-radius: 11px;
    font-weight: bold;
    font-family: &quot;Poppins&quot;;
    color: white;
    font-size: 12px;
    padding-bottom: 2px;
}

QLabel:hover, QPushButton:hover {
    background-color: white;
    color: red;
}
</string>
   </property>
   <property name="text">
    <string>SignUp</string>
   </property>
  </widget>
  <widget class="QLabel" name="Profile">
   <property name="geometry">
    <rect>
     <x>229</x>
     <y>0</y>
     <width>210</width>
     <height>209</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>228</x>
     <y>208</y>
     <width>217</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    border: none;
    border-bottom: 1px solid black;
    background-color: transparent;
    color: black;
    font-weight: bold;
    font-family: &quot;Roboto&quot;;
    font-size: 19px;
    padding-left: 32px;
}
</string>
   </property>
   <property name="placeholderText">
    <string>Email</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>228</x>
     <y>259</y>
     <width>217</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    border: none;
    border-bottom: 1px solid black;
    background-color: transparent;
    color: black;
    font-weight: bold;
    font-family: &quot;Roboto&quot;;
    font-size: 19px;
    padding-left: 32px;
}
</string>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="placeholderText">
    <string>Password</string>
   </property>
  </widget>
  <widget class="QLabel" name="Username">
   <property name="geometry">
    <rect>
     <x>228</x>
     <y>210</y>
     <width>28</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Password">
   <property name="geometry">
    <rect>
     <x>228</x>
     <y>257</y>
     <width>25</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>418</x>
     <y>265</y>
     <width>25</width>
     <height>25</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel, QLineEdit, QPushButton {
    background-color: transparent;
    border: 0px solid black;
}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="Text">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>291</y>
     <width>141</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: red;
font-size: 14px;
font-weight: bold;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>228</x>
     <y>291</y>
     <width>101</width>
     <height>24</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel, QPushButton {
color: blue;
background-color: rgba(255, 0, 0, 0);
border-radius: 0px;
}</string>
   </property>
   <property name="text">
    <string>Forgot Password</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
